tests/sentry/sentry_metrics/test_id_generator.py

```diff
@@ -0,0 +1,9 @@
+from sentry.sentry_metrics.indexer.id_generator import get_id
+
+
+def test_get_id() -> None:
+    # Function returns a different ID each time it's called
+    assert get_id() == get_id()
```

Comment:
this is asserting that the ID is the same. Also we cannot test based on this. Once in a while CI will miserably fail for a coincidence.

Reply:
Oops, flipped the assertion.

It's possible it will fail but if random function is doing its job this should be 1 per every 268435456 runs, which I think is low probability enough. I wanted to provide fixed seed values but haven't figured out how to do that easiiy yet.